packages/flutter_tools/lib/src/commands/channel.dart

```diff
@@ -74,8 +76,10 @@ class ChannelCommand extends FlutterCommand {
         return null;
       },
     );
-    if (result != 0)
-      throwToolExit('List channels failed: $result', exitCode: result);
+    if (result != 0) {
+      throwToolExit('List channels failed: $result\n${rawOutput.join('\n')}',
```

Comment:
Could we guard this behind a verbose flag?  I suspect this could be confusing for some users, and I assume CI will be running verbosely.

Reply:
Fixed, could you take a look once again?

I originally did it without checking verbose flag as git output is only printed if there is an error, and I think it is more confusing for user to see "List channels failed: 128" rather than the actual error message from git.